src/Shared/PackageManagers/NuGetProjectManager.cs

```diff
@@ -221,9 +248,10 @@ private async Task UpdateVersionMetadata(PackageMetadata metadata, NuGetPackageV
             }
 
             string nameLowercase = packageVersionPackageVersionMetadata.Name.ToLowerInvariant();
+            string versionLowercase = metadata.PackageVersion.ToLowerInvariant();
```

Comment:
This lowercasing is not needed for the test cases you added. Do you have an example of when it would be?

Reply:
No, not particularly. I noticed the package version name was lowercased, so I just went ahead and did the same for the version.